src/applications/mhv-medical-records/util/helpers.js

```diff
@@ -461,3 +461,58 @@ export const getLastUpdatedText = (refreshStateStatus, extractType) => {
 
   return null;
 };
+
+/**
+ * @param {Object} nameObject {first, middle, last, suffix}
+ * @returns {String} formatted timestamp
+ */
+
+export const formatNameFirstLast = ({ first, middle, last, suffix }) => {
+  let returnName = '';
+  if (!last) return null;
```

Comment:
This function should never return `null`, because then "null" will print on the report. Is it even possible for the last name to be missing? This is not a case that the original `formatName` function even handled.